scripts/publish_docker_images.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+
+git config user.name "Buildkite on behalf of Wellcome Collection"
+git config user.email "wellcomedigitalplatform@wellcome.ac.uk"
+git remote add ssh-origin "git@github.com:wellcomecollection/weco-deploy.git"
+
+if [[ ! -f RELEASE.md ]]
+then
+  echo "Not publishing because there's no RELEASE.md"
+  exit 0
+fi
+
+# Get the latest metadata about origin/main
+git fetch
+
+HEAD=$(git rev-parse HEAD)
+MAIN=$(git rev-parse origin/main)
+
+git merge-base --is-ancestor HEAD MAIN && ON_MAIN=$? || ON_MAIN=$?
+
+# if (( ON_MAIN == 1 ))
+# then
+#   echo "Not publishing because we're not on main"
+#   exit 0
+# fi
+
+LATEST_TAG=$(git describe --tags --abbrev=0)
+echo "Latest tag is $LATEST_VERSION"
+
+LATEST_TAG="wellcome/weco-deploy:$LATEST_TAG"
+
+MAJOR_TAG=$(echo "$LATEST_VERSION" | tr '.' ' ' | awk '{print $1}')
+MINOR_TAG=$(echo "$LATEST_VERSION" | tr '.' ' ' | awk '{print $2}')
+PATCH_TAG=$(echo "$LATEST_VERSION" | tr '.' ' ' | awk '{print $3}')
+
+VERSION_TAGS="
+  $MAJOR_TAG
+  $MAJOR_TAG.$MINOR_TAG
+  $MAJOR_TAG.$MINOR_TAG.$PATCH_TAG
+  latest"
+
+docker build --tag "$LATEST_TAG" .
+
+docker login --username wellcometravis --password "$DOCKER_HUB_PASSWORD"
+for version in $VERSION_TAGS
+do
+  docker tag "$LATEST_TAG" "wellcome/weco-deploy:$tag"
+  docker push "wellcome/weco-deploy:$tag"
+done
+
+eval $(aws ecr get-login --no-include-email)
+for version in $VERSION_TAGS
+do
+  docker tag "$LATEST_TAG" "760097843905.dkr.ecr.eu-west-1.amazonaws.com/$LATEST_TAG"
+  docker push "760097843905.dkr.ecr.eu-west-1.amazonaws.com/$LATEST_TAG"
+done
+
+aws ecr-public get-login-password --region us-east-1 | xargs -I '{}' docker login --username AWS --password '{}'
```

Comment:
I'm not sure that I agree that this script is any simpler than the Python one (see also `tr '.' ' ' | awk '{print $3}'`), it seems fairly equivalent to me, about the same number of lines modulo comments...

Reply:
The line count may be similar, but the Python implementation is unnecessarily invoked via tox, which has its own Dockerfile, and the code for which is scattered around half a dozen files.

Reply:
We're still using tox though right?

Reply:
Yes, but not for this aspect of the build. I would like to de-tox this repo; I don't think we get much benefit from it.